content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -26,6 +26,13 @@ This permits customization of Istio to operator specific requirements.
 
 1. [Download the Istio release](/docs/setup/kubernetes/getting-started/#downloading-the-release).
 
+    {{< warning >}}
+    These quick-start instruction will not work with a downloaded [istio github repository](https://github.com/istio/istio)
```

Comment:
instruction -> instructions
And uses of github should be GitHub.

Reply:
Done.